docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

```diff
@@ -13,8 +13,8 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
-<img src="../../images/checkboxRadio.png" alt="photo of checkboxRadio in Tizen">
-
+<img src="../../images/checkboxRadio.png" alt="photo of radio in Tizen">
```

Comment:
Is it possible add the  image title? For which is radio button, which is checkbox..

Reply:
@singa2000 I added proper titles